### ISSUE TITLE
Asset path issue in Heroku

### ISSUE THREAD

**Issue by cpalanzo** [apigy/selfstarter#109]
My apologies ahead of time - I'm new to Ruby/Rails - but figured I could just follow the directions and get this running on Heroku.... Almost... 

Seems like the video placeholder file works on my localhost - but NOT when running in Heroku - I know it's something to do with assets - but not sure what. Any guidance would be appreciated.

Here are the working and non working sites with image inspected...

![screen shot 2015-11-24 at 12 59 53 pm](https://cloud.githubusercontent.com/assets/9503630/11375919/e1e0ea5a-92ab-11e5-90f8-2c98f02935c9.png)

![screen shot 2015-11-24 at 1 02 20 pm](https://cloud.githubusercontent.com/assets/9503630/11375933/eac9234e-92ab-11e5-8195-aeccc5386b98.png)

**Comment by edorgeville** [comment]
Yeah, images should be displayed using the `image_path` Rails helper instead of hardcoding `/assets/images/xxx`.
